### ISSUE TITLE
BPF translations

### ISSUE THREAD

**Issue by lar-mar** [DigitalFlow/Xrm-Quick-Edit#31]
As a user I want also be able to translate BPF details in the dashboard. Currently stagenames are not available.

**Comment by fvakiliev** [comment]
Yes! I also subscribe to it. I need to translate the BPF's metadata.

**Comment by lar-mar** [comment]
@DigitalFlow Any plans for this feature?
